### PR TITLE
docs: update AIO code to v15

### PR DESCRIPTION
**Pull request by bob-watson**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
The docs were configured for 15.0.0-rc.0.

## What is the new behavior?
The docs were configured for 15.0.0.

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Review by AndrewKushnir — COMMENTED** [review]
@bob-watson thanks for the update. The change itself looks good, but the AIO payload size is unexpected. We can still go ahead with this change and investigate later if needed.

File: goldens/size-tracking/aio-payloads.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "aio": {
     "uncompressed": {
       "runtime": 4325,
-      "main": 455143,
+      "main": 471785,
```
[the remark this answers is not in the document]
Comment (bob-watson) [reply goldens/size-tracking/aio-payloads.json:5]: I don't have any idea where it grew.
I'm not in a particular hurry, but I wanted to get things sync'd up with v15 before I start writing [too many] more topics.

**Review by AndrewKushnir — APPROVED** [review]

**Review by AndrewKushnir — APPROVED** [review]
Reviewed-for: size-tracking.

File: goldens/size-tracking/aio-payloads.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (AndrewKushnir) [reply goldens/size-tracking/aio-payloads.json:5]: Got it, thanks. Let's see if @dgp1130 or @alan-agius4 may have some ideas on this size increase. Anyway, let's proceed with the change (to make sure no other work is blocked) and investigate the size increase as a followup.

[the remark this answers is not in the document]
Comment (alan-agius4) [reply goldens/size-tracking/aio-payloads.json:5]: We should investigate this as a follow up for sure.

Mind creating a separate issue to keep track of this? I can probably look into this mid or later this week.

**Comment by bob-watson** [comment]
I created https://github.com/angular/angular/issues/48171 to follow up on this.

File: goldens/size-tracking/aio-payloads.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (bob-watson) [reply goldens/size-tracking/aio-payloads.json:5]: https://github.com/angular/angular/issues/48171

[the remark this answers is not in the document]
Comment (clydin) [reply goldens/size-tracking/aio-payloads.json:5]: I spent some time this trying to investigate this.  I tried adding each package one at a time locally to try to reproduce the size increase but was unable to replicate the ~16KB change.  Also opened #48179 to test from CI and was also not able to replicate.  The sizes did go up a small amount (~600 bytes) though but i'm not sure why this PR has the large size change.

[the remark this answers is not in the document]
Comment (alan-agius4) [reply goldens/size-tracking/aio-payloads.json:5]: The culprit is `safevalues`. I explain more about this in https://github.com/angular/angular/issues/48171#issuecomment-1324728758

**Comment by alan-agius4** [comment]
@AndrewKushnir PTAL.

**Review by dylhunn — APPROVED** [review]
reviewed-for: size-tracking

**Comment by alan-agius4** [comment]
Caretaker note: note sure why the mergeablility status is not green.

**Comment by AndrewKushnir** [comment]
@alan-agius4 @clydin huge thanks for the investigation, finding the root cause and the solution! 👍

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit 76e861f8dedd19258f1565df3b26d88cabe0c2a6.

**Pull request closed without merging**